community/sustainability_effort.md

```diff
@@ -0,0 +1,80 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) and in return have their PRs prioritized for review.
+
+Help is also needed for triaging new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels.
+
+## Commitment
+
+The Argo project currently has 4 designated [roles](https://github.com/argoproj/argoproj/blob/main/community/membership.md):
+- Member
+- Reviewer
+- Approver
+- Lead
+
+For those who focus on just one area of the code (such as UI), the `Reviewer` and `Approver` roles can optionally be scoped to just that area.
```

Comment:
```suggestion
The Reviewer and Approver roles can optionally be scoped to an area of the codebase (for example, UI or docs).
```

- more consistent sentence structure with the rest of the doc and simplified a bit [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- the roles do not have backticks elsewhere in the doc, so remove them
- "for example" is a bit easier to understand for foreign speakers than "such as"
  - also added another example of docs